### ISSUE TITLE
Could hammer support generated columns?

### ISSUE THREAD

**Issue by jameszhujian** [daichirata/hammer#38]
Hi there,
We want to add a generated column to our schema, but hammer doesn't like the syntax. The DDL is like:
expiry TIMESTAMP AS (IF(status IS NULL, updated_at, '10000-01-01 00:00:00 UTC')) STORED

Thanks.

James

**Comment by daichirata** [comment]
hammer uses https://pkg.go.dev/cloud.google.com/go/spanner/spansql for parsing DDL.  

spansql can correctly parse the following DDLs.

```go
CREATE TABLE user (
  Id STRING(20) NOT NULL,
  FirstName STRING(50),
  LastName STRING(50),
  Age INT64 NOT NULL,
  FullName STRING(100) AS (ARRAY_TO_STRING([FirstName, LastName], " ")) STORED,
) PRIMARY KEY (Id);
```

But, when I try to parse the next DDL, I seem to get an error.

```go
CREATE TABLE user (
  Id STRING(20) NOT NULL,
  FirstName STRING(50),
  LastName STRING(50),
  Age INT64 NOT NULL,
  AgeAbove18 INT64 AS (IF(Age > 18, Age, NULL)) STORED;
) PRIMARY KEY (Id);
```

Detailed conditions are not known, but https://github.com/googleapis/google-cloud-go/tree/main/spanner/spansql needs to be modified to correct this.

**Comment by neglect-yp** [comment]
I've created an PR for this issue. https://github.com/googleapis/google-cloud-go/pull/6341

**Comment by neglect-yp** [comment]
> You can try hammer v0.5.10, which has been released with added support for IF and IFNULL expressions. It will work fine > if you are not using NULLIF or COALESCE expressions (or other unsupported syntax).

https://github.com/daichirata/hammer/pull/37#issuecomment-1212755632

**Comment by daichirata** [comment]
This change is included in the v0.5.10 release.
